tools/coverage.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Clear coverage data, run tests, then generate a coverage report.
+#
+# Usage:
+#
+#   ./coverage.sh [KAFKA_VERSION [TOX]]
+#
+# KAFKA_VERSION - Version of Kafka to run integration tests against. See
+#                 KAFKA_ALL_VERS in the Makefile for a list of known versions.
+#                 If not provided, only unit tests will be run.
+#
+# TOX           - tox executable to use. Defaults to whatever is available on
+#                 your PATH.
+
+KAFKA_VERSION="$1"
+TOX="${2:-tox}"
+
+if [[ -n $KAFKA_VERSION ]]
```

Comment:
I have to say I prefer the `if [[ <expression> ]]; then` version, but "fine" :-)

Reply:
Sure, I can make it take the argument via the environment.